tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -5552,7 +5553,8 @@ Status ConvertFullyConnectedHelper(OpConverterParams* params,
                                    TRT_TensorOrWeights input_a,
                                    TRT_TensorOrWeights input_b,
                                    bool transpose_a, bool transpose_b,
-                                   bool* converted) {
+                                   bool* converted,
+                                   nvinfer1::ITensor** output) {
```

Comment:
Let's change this to StatusOr<nvinfer1::ITensor*> ConvertFullyConnectedImpl and do not add the result to params->output.